docs/gallery/domain/plot_icephys.py

```diff
@@ -457,6 +458,38 @@
     category="electrodes",
 )
 
+#####################################################################
+# One predefined subcategory column is the stimulus template column in the stimuli table. This column can be used to
```

Comment:
Since this is a pre-defined column, I think it may be more intuitive to add this as as subsection in the `Adding an intracellular recording` section. 

I would also add a brief description on what the purpose of the column is. I.e,. the `stimulus` is the actual stimulus that has been recorded from the system as it was applied to the cell. The `stimulus_template` on the other hand is a template (i.e,. an idealized version) of the waveform that is being used as the stimulus. This is useful, e.g., to validate that the actual stimulus that was applied matches the expected waveform of the template. 

I would also add a brief note at the end about the use of timestamps in stimulus templates. Because they are templates that are being reused across many recordings, the timestamps in the templates are usually not aligned with the recording (and often not the reference time of the file either), but they often just start at 0 and are seen as relative to whatever the time of the application of the stimulus is.

Reply:
I added an `Adding stimulus templates` subsection in the `Adding an intracellular recording` section. I kept it after the `adding custom columns...` section for now since it uses the `add_column` function and `category` arguments that get explained there